app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -251,16 +247,117 @@ class PatientComponent extends Component {
     this.setState({ gender: event.target.value });
   }
 
+  /**
+   * This method handles the validation of the user input, it ensure no age below 0
+   * and above 200 can be use to filter the search. 
+   * 
+   * @param {interger} age This is the age the user inputs
+   * @param {string} identifier This is the ID of the age 
+   * @returns 
```

Comment:
Please can we fix this return statement on this JsDocs, it should return a boolean.